scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -1089,7 +1089,7 @@ class ScalaTestAntTask extends Task {
   // Class to hold data from <jvmarg> elements.
   //
   private class JvmArg {
-    private var value: String = null
+    private var value: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.